docker-build/docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '3.3'
+services:
+  sql_service:
+    build:
+      context: .
+      dockerfile: Dockerfile 
+    image: sql_service
+    restart: unless-stopped
+    depends_on: 
+     - db
+    stdin_open: true
```

Comment:
I tried removing this line to see if it would help me get this to run, but nothing changed. The line is probably fine, I just didn't recognize it from mine so I thought it would be a good starting change to try.